cache/s3cache/README.md

```diff
@@ -0,0 +1,27 @@
+# S3Cache
+
+s3cache is an abstration on top of Amazon Web Services (AWS) Simple Storage Service (S3) which implements the tegola cache interface. To use it, add the following minimum config to your tegola config file:
+
+```toml
+[cache]
+type="s3"   
+bucket="tegola-test-data"
+```
+
+## Properties
+The s3cache config supports the following config properties:
+
+- `bucket` (string): [Required] the name of the S3 bucket to use.
+- `region` (string): [Optoinal] the region the bucket is in. Defaults to 'us-east-1'
```

Comment:
The only issue I see is pretty minor; "Optional" miss-spelled a few times in the README.

Reply:
Good catch! I will push up a fix in just a bit. Looks like this is ready to be merged. Thanks for the review.